lib/bonus.stk

```diff
@@ -2278,18 +2278,17 @@ doc>
            (string=? srfi- (substring feat 0 srfi-len))
            (string->number (substring feat srfi-len feat-len)))))
 
-  (let* ((u (%uname))
-         (utf8? (key-get *%system-state-plist* :use-utf8 #f))
-         (conf  (%stklos-configure))
-         (stklos-git (%stklos-git))
-         (commit (key-get stklos-git :commit ""))
-         (srfis  (filter-map %feature->srfi (features)))
-         (git-alist (if (= (string-length commit) 0)
-                        '()
-                        `((build.git.tag ,(key-get stklos-git :tag '()))
-                          (build.git.branch ,(key-get stklos-git :branch '()))
-                          (build.git.commit ,(key-get stklos-git :commit '()))
-                          (build.git.modified ,@(key-get stklos-git :modified '()))))))
+  (define (prop name plist plist-key)
```

Comment:
The `bonus.stk` module contains bits from different SRFIs and also other extra Scheme procedures. `prop` won't be user-visible, because `bonus.stk` includes it inside  a module, but maybe it would make sense to name this `SRFI-176-prop`, just to be a bit clearer?  (Or maybe I'm being pedantic -- feel free to ignore me)

Reply:
It's an internal define. Therefore it should only be visible inside the `version-alist` procedure, and not to users.

Reply:
Ops, sorry. From the github UI it wasn't clear.